packages/editor/src/client/base.tsx

```diff
@@ -29,55 +29,59 @@ import {
   UserRoleListRoute
 } from './route'
 
+import {useTranslation} from 'react-i18next'
+
 export interface BaseProps {
   children?: ReactNode
 }
 
 export function Base({children}: BaseProps) {
   const {current} = useRoute()
 
+  const {t} = useTranslation()
+
   return (
     <NavigationTemplate
       navigationChildren={
         <>
           <RouteMenuLinkButton
             icon={MaterialIconTextFields}
-            label="Articles"
+            label={t('Articles')}
             route={ArticleListRoute.create({})}
             active={current?.type === RouteType.ArticleList}
           />
 
           <RouteMenuLinkButton
             icon={MaterialIconInsertDriveFileOutlined}
-            label="Pages"
+            label={t('Some Pages')}
```

Comment:
some pages?

Reply:
I applied that from the template you made; I thought this was for a particular reason that you named it "some pages" :) Switched it back to "Pages" now